android/app/src/main/java/org/fossasia/openevent/viewmodels/FeedFragmentViewModel.java

```diff
@@ -0,0 +1,55 @@
+package org.fossasia.openevent.viewmodels;
+
+import android.arch.lifecycle.ViewModel;
+import android.content.res.Resources;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.api.APIClient;
+import org.fossasia.openevent.data.facebook.FeedItem;
+import org.fossasia.openevent.utils.ConstantStrings;
+import org.fossasia.openevent.utils.SharedPreferencesUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.schedulers.Schedulers;
+
+public class FeedFragmentViewModel extends ViewModel {
+
+    private List<FeedItem> feedLive;
```

Comment:
This list and all the operations of add and clear should remain in the Fragment itself